importer/LookupTable.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Class to fetch online lookup table.
+
+Works with tables like this:
+https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/se-arbetsl_(sv)/types
+
+Download the table and turn it into a dictionary
+that can then be loaded into the Monument object.
+"""
+import pywikibot as pwb
+import mwparserfromhell as mwp
+
+BASE_URL = "Wikidata:WikiProject WLM/Mapping tables/"
+
+
+class LookupTable(object):
+    """A lookup table converted from wikitext."""
+
+    def filter_tr(self, node):
+        return isinstance(node, mwp.nodes.tag.Tag) and node.tag == 'tr'
+
+    def filter_td(self, node):
+        return isinstance(node, mwp.nodes.tag.Tag) and node.tag == 'td'
+
+    def fetch_page(self, tablename):
+        """Download lookup table from given path."""
+        site = pwb.Site("wikidata", "wikidata")
+        return pwb.Page(site, BASE_URL + tablename)
+
+    def get_page_metadata(self, page):
+        """Get permalink and last changed timestamp of wiki page."""
+        timestamp = page.editTime().isoformat()
+        permalink = page.permalink()
+        return {'permalink': permalink, 'timestamp': timestamp}
+
+    def table_to_json(self, tablename):
```

Comment:
It would be nice with an example (or test) showing what the source data looks like.

For testing/clarity it might be easier to split this into two functions where one takes `page.get()` as it's input and returns `lookup_dict["mappings"].` Then you could test it easily without needing to fetch a live page.